### PR TITLE
[Quickfix] Correcting BS3 regression on badge color + spelling mistake

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/views/conversations/_conversation.haml [hunk 1]
```diff
@@ -16,10 +16,10 @@
               = other_participants.count - 1
 
       .bg
-        .badge.badge-dafault.message_count
+        .badge.badge-default.message_count
           = conversation.messages.size
         - if visibility.unread > 0
-          .badge.badge-important.unread_message_count
+          .badge.progress-bar-danger.unread_message_count
```
Comment (svbergerem) [inline app/views/conversations/_conversation.haml]: I don't think we should misuse Bootstrap classes. Instead you could set a red background for `.badge-important` via CSS.

**Comment by svbergerem** [comment]
We have the same spelling mistake in `_conversation_subject.haml`. Would you mind fixing that too?

**Comment by AugierLe42e** [comment]
Sure!

**Comment by AugierLe42e** [comment]
Wow... Lots of random fails. Travis must be busy.

**Comment by svbergerem** [comment]
Thank you.

**Pull request merged**